examples/lammps/test.bats

```diff
@@ -37,7 +37,8 @@ load "${CHTEST_DIR}/common.bash"
 setup () {
     scope full
     prerequisites_ok "$ch_tag"
-    multiprocess_ok
+    multiprocess_ok "$ch_tag"
```

Comment:
Does `multiprocess_ok` ever get a value that is not `$ch_tag`? If that's correct, it should just continue using the global variable, no?

Reply:
Agreed, missed this on the recent rounds of commits. Will revert.